_locales/en/messages.json

```diff
@@ -331,11 +343,20 @@
   },
   "issues": {
     "message": "Issues",
-    "description": "Label for the CSSLint issues block on the style edit page"
+    "description": "Label for the CSS linter issues block on the style edit page"
   },
   "issuesHelp": {
-    "message": "The issues found by <a href='https://github.com/CSSLint/csslint' target='_blank'>CSSLint</a> with these rules enabled:",
-    "description": "Help popup message for the CSSLint issues block on the style edit page"
+    "message": "The issues found by $link$ rules:",
+    "description": "Help popup message for the selected CSS linter issues block on the style edit page",
+    "placeholders": {
+      "link": {
+        "content": "$1"
+      }
+    }
+  },
+  "stylelintConfig": {
```

Comment:
Simply "Configure rules" - this message is used for CSSLint too.

It'd be reasonable to group all linter related messages via `linter` prefix: linterConfigRules, linterAllRulesLink, linterRulesTitle, linterConfigParseError, linterConfigInvalidRulesError, linterCSSLintLegend

Reply:
er, linterConfigTitle